user-guide/Cloud_Platform/CloudAdminApp/Cloud_connection_verification.md

```diff
@@ -6,7 +6,7 @@ uid: CloudConnectionVerification
 
 To make sure you can get the most out of having your DataMiner System connected to dataminer.services, you need to get your organization verified on dataminer.services. This verification ensures that you have access to all the current and upcoming new dataminer.services features.
 
-Current benefits include access to the Catalog in order to install protocols directly from the dataminer.services using the artifact deployer (available starting from Cloud Pack v2.4).
+Benefits include access to the licensed connectors in the Catalog for systems with a perpetual license. The Catalog allows you to [install connectors directly](xref:Deploying_a_catalog_item) from dataminer.services using the artifact deployer (available starting from Cloud Pack v2.4). Systems with a subscription license can access all connectors.
```

Comment:
@baptistepa, do we still need to mention the artifact deployer and the starting Cloud Pack version here? This doesn't seem all that relevant anymore, so we could just simplify this to:
 ```suggestion
Benefits include access to the licensed connectors in the Catalog for systems with a perpetual license. The Catalog allows you to [install connectors directly](xref:Deploying_a_catalog_item) from dataminer.services. Systems with a subscription license can access all connectors.
```

Reply:
No this seems fine for me.